testplan/common/entity/base.py

```diff
@@ -1498,7 +1505,7 @@ def run(self):
             raise self._runnable.result
         return self._runnable.result
 
-    def _handle_abort(self, signum):
+    def _handle_abort(self, signum, frame):
```

Comment:
what is frame for?

Reply:
That is expected from a signal handler: https://docs.python.org/3/library/signal.html#signal.signal